graylog2-web-interface/src/components/users/UsersOverview/UserOverviewItem/ActionsCell.jsx

```diff
@@ -50,9 +52,17 @@ const ReadOnlyActions = ({ user }: { user: UserOverview }) => {
 };
 
 const EditActions = ({ user, user: { username, id, fullName, accountStatus, external, readOnly } }: { user: UserOverview }) => {
+  const currentUser = useContext(CurrentUserContext) || {};
+
   const _toggleStatus = () => {
-    const newStatus = accountStatus === 'enabled' ? 'disabled' : 'enabled';
-    UsersDomain.setStatus(id, newStatus);
+    // eslint-disable-next-line no-alert
+    if (accountStatus === 'enabled' && window.confirm(`Do you really want to disable user ${fullName}? All current sessions will be terminated.`)) {
+      UsersDomain.setStatus(id, 'disabled');
+
+      return;
+    }
+
+    UsersDomain.setStatus(id, 'enabled');
```

Comment:
When disabling a user and closing the confirm dialog, `UsersDomain.setStatus(id, 'enabled');` is being called and the API responds with `Error: Not Modified`